website/blog/2020-11/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Gardener v1.11 and v1.12 Released
+linkTitle: Gardener v1.11 and v1.12
+newsSubtitle: November 4, 2020
+type: blog
+publishdate: 2020-11-04
+archivedate: 2020-12-31
+authors:
+- name: Tim Usner
+  email: tim.usner@sap.com
+  avatar: https://avatars2.githubusercontent.com/u/40451181?s=460&u=4df34635cf86b924700ef4152ec8462eeaa35721&v=4
+aliases: ["/blog/2020/11/04/00"]
+---
+
+Two months after our last Gardener release update, we are happy again to present release v1.11 and v1.12 in this blog post. Control plane migration, load balancer consolidation, new security features are just a few topics we progressed with. As always, a detailed list of features, improvements, and bug fixes can be found in the [release notes](https://github.com/gardener/gardener/releases) of each release. If you are going to update from a previous Gardener version, please take your time to go through the action items in the release notes.
+
+## Noteable Changes in v1.12
+
+Release v1.12, fresh from the oven, is shipped with plenty of improvements, features and some API changes we want to pick up in the next sections. 
+
+### Functionless DNS Providers ([gardener/gardener#3036](https://github.com/gardener/gardener/pull/3036))
+
+This release drops the support for so-called functionless DNS providers. Those are providers in a shoot’s specification (`.spec.dns.providers`) which don’t serve the shoot’s domain (`.spec.dns.domain`), but are created by Gardener in the seed cluster to serve DNS requests coming from the shoot cluster. If such providers don’t specify a `type` or `secretName` the creation or update request for the corresponding shoot is denied.
+
+### Seed Taints ([gardener/gardener#2955](https://github.com/gardener/gardener/pull/2955))
+
+With the introduction of `seed.spec.settings`, we spent the seed object a dedicated section in favor of previously used taints `disable-capacity-reservation, disable-dns, invisible`. These already deprecated taints were still considered and synced for a certain time which allowed operators to switch their integration to the new `settings` field. As of version v1.12, support for the mentioned taints has been dropped and are automatically removed from seed objects. You may use the actual taint names in a future release of Gardener again to serve your own use-cases.
+
+### Load balancer events during shoot reconciliation ([gardener/gardener#3028](https://github.com/gardener/gardener/pull/3028))
```

Comment:
```suggestion
### Load Balancer Events During Shoot Reconciliation ([gardener/gardener#3028](https://github.com/gardener/gardener/pull/3028))
```

(same for other headlines)

Reply:
Pretty inconsistent to https://github.com/gardener/documentation/pull/126 headlines. Should we also introduce a capitalization policy 🤯